src/olympia/reviewers/views.py

```diff
@@ -1320,7 +1320,9 @@ def get_addon_object(self):
         return self.addon_object
 
     def get_version_object(self):
```

Comment:
This isn't technically new, but since you made changes to it... I can't find any test hitting this method. When is it supposed to be called, do you know ?

Reply:
It is supposed to get called when a serializer whose instance is not a version, needs to use the mixin. I found this use case via tests, so maybe those tests are faulty if this never needs to be called in real world scenarios. I did have some concerns about what the DraftCommentSerializer is doing.